server/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1
+FROM python:3-slim
+WORKDIR /app
+COPY requirements.txt ./
+RUN pip3 install --no-cache-dir -r requirements.txt
+COPY ./ .
+EXPOSE 5000
```

Comment:
I think EXPOSE is useful if we are using multi-containers application. One benefit of using the EXPOSE instruction, whether you are running a multi-container application or not, is that it helps others understand what port the application listens by just reading the Dockerfile without the need of going through your code.

Reply:
I wasn't 100% sure if it was necessary but I liked that it makes people understand what ports it's running on easier